src/Analysis/Ast/Impl/Types/PythonUnionType.cs

```diff
@@ -42,15 +44,11 @@ internal sealed class PythonUnionType : LocatedMember, IPythonUnionType {
         #region IPythonType
 
         public string Name {
-            get {
-                lock (_lock) {
-                    return CodeFormatter.FormatSequence("Union", '[', _types.ToArray());
-                }
-            }
+            get { lock (_lock) { return CodeFormatter.FormatSequence("Union", '[', _types.ToArray()); } }
```

Comment:
I can't say that it is easier to read one line.

Reply:
OK